src/openzaak/config/forms.py

```diff
@@ -0,0 +1,31 @@
+import socket
+from urllib.parse import urlparse
+
+from django.forms import ModelForm, ValidationError
+from django.utils.translation import ugettext_lazy as _
+
+from .models import NLXConfig
+
+
+class NLXConfigForm(ModelForm):
+    class Meta:
+        model = NLXConfig
+        fields = ("directory", "outway")
+
+    def clean_outway(self):
+        outway = self.cleaned_data["outway"]
+
+        if not outway:
+            return outway
+
+        # try to tcp connect to the port
+        parsed = urlparse(outway)
+        with socket.socket() as s:
+            try:
+                s.connect((parsed.hostname, parsed.port))
```

Comment:
I'd like to see a timeout specified, 2s for example should be enough: https://docs.python.org/3.6/library/socket.html#timeouts-and-the-connect-method

Reply:
timeout is added